src/ripple/json/impl/json_reader.cpp

```diff
@@ -29,6 +29,8 @@ namespace Json
 // Implementation of class Reader
 // ////////////////////////////////
 
+constexpr unsigned nest_limit = 1000;
```

Comment:
Just as a point of reference, the code that converts JSON to STObjects/STArrays limits recursion (objects containing objects or arrays and on down) to 64: https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/STParsedJSON.cpp#L693.  Not suggesting any change here.

Reply:
Might worth making a public static member that can be referenced externally, like a unit test.

Reply:
Such a unit test would get pretty ugly.  Not sure it is worth it.